### DISCUSSION TITLE
Mobile UI

### DISCUSSION THREAD

**Discussion by DevrayePatil** [Budibase/budibase#7318] [Help] [answered]
Can we create different UI based on screen width as like mobile screen

**Comment by andz-bb** [comment, marked as answer]
Hey @DevrayePatil - a good way to alter the UI based on the device size is by using conditions on components. You can select device size as a condition and then conditionally render components based on that. You can also change individual settings on a given component based on these conditions.

Here's a quick example where I am changing the text and icon based on the device size condition. I also then set the mobile-only component to hide on larger screen sizes.

![quick conditions demo](https://user-images.githubusercontent.com/110921612/185102114-5846756f-f25a-4822-86fa-2d99c8062136.gif)

<img width="1278" alt="device size conditions" src="https://user-images.githubusercontent.com/110921612/185102382-9da3dedf-4a47-497b-9ffb-de6b52e3ee83.png">



You can see more details about this[ in our documentation here.](https://docs.budibase.com/docs/designing-for-mobile#key-responsive-settings-and-defaults) Hope this has been helpful for you, let me know if you run into any problems with it 👍
